README.md

```diff
@@ -18,25 +18,45 @@ If you have [Vagrant](https://www.vagrantup.com) installed and want to get
 developing or playing right away, just run `vagrant up` and a few minutes
 later [you'll have a working server](https://localhost:4443/myradio/).
 
-During setup, you'll be asked for database credentials - you can use:
-* Hostname: localhost
-* Database: myradio
-* Username: myradio
-* Password: myradio
-
-Note that the Vagrant bootstrap script gives the myradio user CREATEDB
-permissions, so be sure to never run this in a production environment, or remove
-the permission before doing so.
+Steps
+-----
+Items in [Square Brackets] give notes and advice if things don't work
+
+INSTALL:
+ - Make sure you have both "vagrant" and "virtualbox" installed and configured
+ - [Try reinstalling vagrant after virtualbox, or downloading an installer]
+ - Clone myradio, open the folder in console and run "vagrant up"
+ - [This may take a long time and use up all processing power]
+
+CONNECT:
+ - Open up "https://localhost:4443/myradio/" in a browser
+ - [Use Chrome as this often fails to run on Firefox]
+ - It will say "connection not private" so press "advanced" and then "proceed"
+
+DATABASE:
+ - On the intro screen press "Click here to continue"
+ - Enter the database details (see Database Credentials) and press Next
+ - Enter the database details AGAIN and press next
```

Comment:
Gah, i'll have a debug on this. I would love this not to be a thing.

Reply:
It's not all that bad a problem, even if you aren't aware of it when going in. If it's not a rather simple fix I'm not sure there's all that much point

Reply:
Ok, i've now fixed this bug in 8372a4d, please remove this line 👍